main/src/main/java/com/airbnb/reair/incremental/ReplicationServer.java

```diff
@@ -185,6 +188,8 @@ public ReplicationServer(
 
     this.jobExecutor = new ParallelJobExecutor("TaskWorker", numWorkers);
     this.copyPartitionJobExecutor = new ParallelJobExecutor("CopyPartitionWorker", numWorkers);
+    this.auditLogBatchSize = conf.getInt(
+        ConfigurationKeys.AUDIT_LOG_PROCESSING_BATCH_SIZE, 128);
```

Comment:
Should we go for a lower default batch size? If the performance with say 32 is good enough, then it might be preferable to avoid very large queries.

Reply:
Sure, makes sense. 32 had pretty good performance too.